manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -46,6 +47,7 @@ linux:
     repository: registry.k8s.io/sig-storage/csi-node-driver-registrar
     tag: v2.8.0
     pullPolicy: IfNotPresent
+    #digest: sha256:
```

Comment:
Move this one line to be after `tag` consistently in this file

Reply:
done